core/models.py

```diff
@@ -1,3 +1,35 @@
 from django.db import models
 
-# Create your models here.
+
+
+class client(models.Model):
```

Comment:
In python the general convention is to use CamelCase for class names. You can google camelcase and see how it works

In this situation it would be 
```
class Client(models.Model):
...
```